egs2/librispeech_100/asr1_biasing/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+export PATH=~/rds/rds-t2-cs164-KQ4S3rlDzm8/gs534/MultiModal/espnet/tools/sctk/bin/:$PATH
```

Comment:
This PATH contains some local information. I'm not sure. Probably sctk does not require particular `export`.

Reply:
My local env is a bit complicated so I will remove it once the scoring with biasing words is done. Thanks!